ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master__nfd.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: build/Dockerfile
+  from: os
+  to: ci-artifacts
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.8"
+resources:
+  '*':
+    limits:
+      memory: 2Gi
+    requests:
+      cpu: 2000m
+      memory: 2Gi
+tests:
+- as: e2e-master
+  cron: 0 */23 * * *
```

Comment:
in the general organization of the files / tests, I chose to:
1. not to run any periodic test in master`, only `presubmit` (so you can remove this `cron` stanza to follow the same pattern)
2. in `release-4.x`, run the nightly tests dedicated to OCP `4.x`
so this `4.8` testing should be moved to `release4.8__nfd`